### PR TITLE
add content-trust commands: show, update

### PR DESCRIPTION
**Pull request by ankurkhemani**
Add commands to configure registry policies, currently adding content-trust.

```
az acr config content-trust show
az acr config content-trust update --status enabled/disabled
```
 
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

File: src/command_modules/azure-cli-acr/setup.py [hunk 1]
```diff
@@ -34,7 +34,7 @@
     'azure-mgmt-resource==2.0.0',
     'azure-mgmt-storage==1.5.0',
     'azure-storage-blob==1.1.0',
-    'azure-mgmt-containerregistry==2.0.0',
+    'azure-mgmt-containerregistry==2.1.0',
```
Comment (yugangw-msft) [inline src/command_modules/azure-cli-acr/setup.py]: Can you update the version here? otherwise the CI won't succeed.

Comment (ankurkhemani) [reply src/command_modules/azure-cli-acr/setup.py]: Done.

File: src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/commands.py [hunk 2]
```diff
@@ -40,6 +41,11 @@ def load_command_table(self, _):  # pylint: disable=too-many-statements
         client_factory=cf_acr_registries
     )
 
+    acr_policy_util = CliCommandType(
+        operations_tmpl='azure.cli.command_modules.acr.policy#{}',
+        client_factory=cf_acr_registries_stable
```
Comment (djyou) [inline src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/commands.py]: `table_transformer`?

File: src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/policy.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+                                  resource_group_name=None):
+    _, resource_group_name = validate_managed_registry(
+        cmd.cli_ctx, registry_name, resource_group_name, POLICIES_NOT_SUPPORTED)
+    return client.list_policies(resource_group_name, registry_name).trust_policy
+
+
+def acr_config_content_trust_update(cmd,
+                                    client,
+                                    registry_name,
+                                    status,
```
Comment (djyou) [inline src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/policy.py]: Default `status` to None

File: src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/policy.py [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+                                    resource_group_name=None):
+    _, resource_group_name = validate_managed_registry(
+        cmd.cli_ctx, registry_name, resource_group_name, POLICIES_NOT_SUPPORTED)
+
+    trust_policy = client.list_policies(resource_group_name, registry_name).trust_policy
+    if trust_policy.status != status:
```
Comment (djyou) [inline src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/policy.py]: Can we remove this client side logic? Basically as long as `status` is not `None`, we should send the request and let the server decide what the updated policy is.

File: src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_constants.py [hunk 5]
```diff
@@ -16,3 +17,5 @@
 
 CLASSIC_REGISTRY_SKU = [SkuName.classic.value]
 MANAGED_REGISTRY_SKU = [SkuName.basic.value, SkuName.standard.value, SkuName.premium.value]
+
+POLICY_STATUS = [PolicyStatus.enabled, PolicyStatus.disabled]
```
[the remark this answers is not in the document]
Comment (ankurkhemani) [reply src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_constants.py]: No

File: src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py [hunk 6]
```diff
@@ -86,6 +87,9 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
         c.argument('repository', help='The repository name to do a manifest-only copy for images.', action='append')
         c.argument('force', help='Overwrite the existing tag of the image to be imported.', action='store_true')
 
+    with self.argument_context('acr config content-trust') as c:
+        c.argument('status', help="Indicates whether content-trust is enabled or disabled.", arg_type=get_enum_type(POLICY_STATUS))
+
```
Comment (djyou) [inline src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py]: You can 
```
from azure.mgmt.containerregistry.v2018_02_01_preview.models import PolicyStatus
arg_type=get_enum_type(PolicyStatus)
```
and remove `POLICY_STATUS` from constants.

File: src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_constants.py (hunk 5 above)
[the remark this answers is not in the document]
Comment (djyou) [reply src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_constants.py]: I don't think the array is needed here.

**Comment by ankurkhemani** [comment]
@yugangw-msft Ready to merge. Thanks!

**Pull request merged**